src/merge_default_graph.rs

```diff
@@ -0,0 +1,20 @@
+use anyhow::Result;
+use sophia::api::{quad::Quad, source::QuadSource};
+
+use crate::common::{pipe::PipeSubcommand, quad_handler::QuadHandler};
+
+/// Merge named graphs with the default graph
+#[derive(clap::Args, Clone, Debug)]
+pub struct Args {
+    #[command(subcommand)]
+    pipeline: Option<PipeSubcommand>,
+}
+
+pub fn run<Q: QuadSource>(quads: Q, args: Args) -> Result<()>
+where
+    <Q as QuadSource>::Error: Send + Sync,
+{
+    log::trace!("merge-default-graph args: {args:#?}");
+    let handler = QuadHandler::new(args.pipeline);
+    handler.handle_triples(quads.to_triples())
```

Comment:
question: compilation raises stackoverflow errors due to this line. I'm not sure I understand what is the expected API usage here

```
error[E0275]: overflow evaluating the requirement `TripleToQuad<ToTriples<TripleToQuad<ToTriples<TripleToQuad<ToTriples<TripleToQuad<...>>>>>>>: QuadSource`
```